services/account_type/cloudtrail_owner.yaml

```diff
@@ -0,0 +1,114 @@
+AWSTemplateFormatVersion: "2010-09-09"
```

Comment:
Also, as a general convention for CFN templates, I try to follow [these YAML conventions](https://stackoverflow.com/questions/19109912/do-i-need-quotes-for-strings-in-yaml#answer-22235064), i.e.
- No quotes around strings unless you need to
- You need to quote strings if there are special characters, you need to coerce a string, or if quoting an ambiguous key
- If you do need to quote strings, prefer single quote `'` because so don't have to escape